### ISSUE TITLE
Ggplot graphics truncated

### ISSUE THREAD

**Issue by piccolbo** [nteract/hydrogen#676]
**Description:**

Ggplot for python graphics excludes x axis

**Steps to Reproduce:**

1. [First Step]

Load ggplot
2. [Second Ste

Plot a graph

Sorry on a phone now can provide 4 line example

**Versions:**

Which OS and which version of Hydrogen and Atom are you running?
 Os x Sierra  hydrogen 1.8,0

You can get this information from copy and pasting the output of `atom --version`  from the command line.

**Logs:**

Please post any error logs and the output of the developer tools as described in our [Debugging Guide](https://github.com/nteract/hydrogen/blob/master/TROUBLESHOOTING.md).

**Comment by BenRussert** [comment]
@piccolbo can you be more specific? I ran some [ggplot code just now](https://youtu.be/-l7cuhKtlvA?t=8) and saw no issues.

**Comment by piccolbo** [comment]
You bet!
```
import pandas as pd
import ggplot as gg
df = pd.DataFrame(dict(x = range (1,10), y = range(1,10)))
gg.ggplot(df, gg.aes(x='x', y = 'y')) + gg.geom_line()
```
<img width="1239" alt="screen shot 2017-03-29 at 9 59 43 pm" src="https://cloud.githubusercontent.com/assets/905563/24488368/3263623a-14cb-11e7-8a8f-75ebc944435d.png">

After my first report I discovered that if I select the gplot line and the following line, with a random harmless expression, and then evaluate, then the problem is lessened. But the graphics panel is still too small -- it scrolls though. 

```
print(gg.ggplot(df, gg.aes(x='x', y = 'y')) + gg.geom_line())
1
````

<img width="834" alt="screen shot 2017-03-29 at 10 05 32 pm" src="https://cloud.githubusercontent.com/assets/905563/24488465/ddf8b0fa-14cb-11e7-8fbf-a7af2dcc564b.png">

**Comment by BenRussert** [comment]
Looks like this is the same isssue as #632.  We are talking about an enhancement to resize images there.

I'll close this issue, but feel free to reply here or in #632 with any questions.

**Comment by piccolbo** [comment]
I agree there's a big overlap with#632. But mostly for my education, when is the decision between inline and block graphics made? How can such a big inline bubble that covers the rest of the code make sense? Why no scroll bars if part of the graphics is hidden? My answer, coming from rstudio, would be never inline and always size the bubble big enough to avoid scrolling

**Comment by lgeiger** [comment]
That's definitely a bug.
It looks like we try to display the plot as a inline result bubble (no scrollbars...) since ggplot sends a small plot id (`<ggplot: (12345)>`) after the plot.

The relevant lines are [here](https://github.com/nteract/hydrogen/blob/master/lib/result-view.js#L162).
If there isn't a easy fix, I would suggest we fix it during #655.
